src/index.js

```diff
@@ -1,8 +1,7 @@
-let isString = value => typeof value === 'string' || value instanceof String;
+const isString = (value) => typeof value === 'string' || value instanceof String;
 
-
-let scanClojure = content => {
-    var a = [];
+const scanClojure = (content) => {
+    const a = [];
```

Comment:
Wouldn't const a = [] imply that it is immutable, and so can't be mutated?

Reply:
It’s more like `final` in Java, if you’re familiar. It can’t be redeclared or reassigned, but you can mutate the array.